src/eckit/io/PooledFile.cc

```diff
@@ -21,12 +21,31 @@
 #include "eckit/io/PooledFile.h"
 #include "eckit/log/Bytes.h"
 
```

Comment:
missing includes map, mutex, utility, sstream
remove thread
move #include "eckit/io/PooledFile.h" to the top of the include list